connect/src/spirc.rs

```diff
@@ -1081,13 +1080,13 @@ impl SpircTask {
         self.resolve_context(&mut ctx).await?;
         self.connect_state.update_context(ctx);
 
-        self.connect_state.player.next_tracks.clear();
+        self.connect_state.next_tracks.clear();
         self.connect_state.player.track = MessageField::none();
 
         let index = cmd.playing_track_index as usize;
         self.connect_state.set_shuffle(cmd.shuffle);
         if cmd.shuffle {
-            self.connect_state.set_current_track(index)?;
+            self.connect_state.set_current_track(index, false)?;
```

Comment:
That `bool` here - was it a remnant from the past? I remember we had something like it in `spirc` to indicate whether we should update the state or not. Never was a fan of it, because it's not very clear what the argument does.

Reply:
Same as above, decides if the current track should be set from the current or shuffle context. Will probably change soon. And yeah especially from a code review perspective this is horrible. Will see to improve these parts^^